kategory-annotations-processor/build.gradle

```diff
@@ -6,7 +6,7 @@ apply plugin: 'kotlin-kapt'
 dependencies {
     compile "org.jetbrains.kotlin:kotlin-stdlib-jre8:$kotlinVersion"
     compile project(':kategory-annotations')
-    compile 'com.squareup:kotlinpoet:0.4.0'
+//    compile 'com.squareup:kotlinpoet:0.4.0'
```

Comment:
????

Reply:
It was commented out before I added Lens generation but I moved away from KotlinPoet in this PR so I commented it again. Should I go ahead and remove it?

Reply:
Yes, definitely :D What was the issue with Kotlinpoet?

Reply:
KotlinPoet wasn't handling generics and imports correctly. Handling that manually with Eugenio's lib was easy so I fully moved away from KotlinPoet.